bin/src/MixinGenerator.php

```diff
@@ -262,7 +262,7 @@ private function applyTypeTemplate(string $type, string $typeTemplate): string
     {
         $combinedType = sprintf($typeTemplate, $type);
 
-        if ('null|empty' === $combinedType) {
+        if ('empty|null' === $combinedType) {
```

Comment:
Same here?

Reply:
The code generator now needs to generate `|null` to match php-cs-fixer expectations